test/algorithms/buffer/read_from_wkt_file.hpp

```diff
@@ -38,8 +38,6 @@ inline std::string read_from_wkt_file(std::string const& filename)
         out << std::fixed << std::setprecision(20) << bg::wkt(mp);
     }
 
-    BOOST_CHECK(! out.str().empty());
```

Comment:
Same comment as below, this change to silently ignore a missing input seems dangerous in a test context.